FluidNC/src/Config.h

```diff
@@ -23,26 +23,26 @@ Some features should not be changed. See notes below.
 // machine.h is #included below, after some definitions
 // that the machine file might choose to undefine.
 
-const int MAX_N_AXIS = 6;
+const int MAX_N_AXIS = 9;
 
 const int MAX_MESSAGE_LINE = 256;
 
 // Axis array index values. Must start with 0 and be continuous.
 // Note: You set the number of axes used by changing MAX_N_AXIS.
-// Be sure to define pins or servos in the machine definition file.
 const int X_AXIS = 0;  // Axis indexing value.
 const int Y_AXIS = 1;
 const int Z_AXIS = 2;
 const int A_AXIS = 3;
 const int B_AXIS = 4;
 const int C_AXIS = 5;
+const int U_AXIS = 6;
+const int V_AXIS = 7;
+const int W_AXIS = 8;
+
+const std::unordered_set<int> LINEAR_AXES = { X_AXIS, Y_AXIS, Z_AXIS, U_AXIS, V_AXIS, W_AXIS };
+const std::unordered_set<int> ROTARY_AXES = { A_AXIS, B_AXIS, C_AXIS };
 
```

Comment:
These unordered_set definitions in a file that is included by nearly ever .cpp file increases the size of the image by 60K.  We cannot afford that.